test/cppsuite/test_harness/workload/database_operation.cxx

```diff
@@ -49,16 +49,14 @@ populate_worker(thread_context *tc)
          * is closed, WiredTiger APIs close the cursors too.
          */
         scoped_cursor cursor = tc->session.open_scoped_cursor(coll.name.c_str());
-        for (uint64_t i = 0; i < tc->key_count; ++i) {
-            /* Start a txn. */
+        uint64_t j = 0;
+        while (j < tc->key_count) {
             tc->transaction.begin();
-            if (tc->insert(cursor, coll.id, i)) {
-                /* We failed to insert, rollback our transaction and retry. */
-                tc->transaction.rollback();
-                --i;
-                continue;
+            if (tc->insert(cursor, coll.id, j)) {
+                if (tc->transaction.commit()) {
```

Comment:
You don't need the curly brackets here.

Reply:
I thought we decided we were always doing curly brackets cause we're now doing C++ style?

Reply:
True, my bad.